Source/TailwindTraders.ShippingManagement/Services/AnalysisService.cs

```diff
@@ -39,7 +39,7 @@ private async Task<string> TrainModelAsync()
                 {
                     CustomFormModel result = await _formTrainingClient.StartTrainingAsync(new Uri(_settings.FormRecognizedTrainningDataUrl), useTrainingLabels: false).WaitForCompletionAsync();
 
-                    CustomFormModel model = await _formTrainingClient.GetCustomModelAsync(result.ModelId);
+                    await _formTrainingClient.GetCustomModelAsync(result.ModelId);
```

Comment:
`GetCustomModelAsync` returns a `CustomFormModel ` which in this case is not being assigned, so this call is doing nothing.

If all you want is to return the `result.ModelId` you already have it and you can return it :) 

```
CustomFormModel result = await _formTrainingClient.StartTrainingAsync(new Uri(_settings.FormRecognizedTrainningDataUrl), useTrainingLabels: false).WaitForCompletionAsync();
return result.ModelId;
```

Reply:
@maririos    I have update as your comments.